pulp_container/app/tasks/sync_stages.py

```diff
@@ -190,7 +190,7 @@ def handle_blobs(self, man, content_data, total_blobs):
         for layer in content_data.get("layers") or content_data.get("fsLayers"):
             if not self._include_layer(layer):
                 continue
-            blob_dc = self.create_blob(man, layer)
+            blob_dc = self.create_blob(man, layer, self.deferred_download)
```

Comment:
It's kind of weird, that the special handling is performed at the normal blob, and not the special config blob.
How about you specify `deferred_download=False` explicitly for config blobs?
I'll try to outline the rest of this solution in suggestions:
```suggestion
            blob_dc = self.create_blob(man, layer)
```

Reply:
I cannot add the corresponding suggestion to line 198

Reply:
special handling is left for normal blobs because the user can actually make a call whether he wants it downloaded or not. He can control that by specifying the download policy in the remote. Whilst config blob he cannot affect in any case because it will always be downloaded.
I can put another the boolean evaluation into the create_blob if that improves readability.

Reply:
In the Pulp mindset, i consider honoring the download_policy to be 'normal'. Therefore overwriting it for certain content classes would be the special case.